thsvs_simpler_vulkan_synchronization.h

```diff
@@ -817,6 +815,7 @@ void thsvsGetVulkanImageMemoryBarrier(
         const ThsvsVkAccessInfo* pPrevAccessInfo = &ThsvsAccessMap[prevAccess];
 
 #ifdef THSVS_ERROR_CHECK_POTENTIAL_HAZARD
+        assert(prevAccess < THSVS_NUM_ACCESS_TYPES); // Make sure the lookup is in range

```

Comment:
This should probably be in a new #define - maybe something like `THSVS_ERROR_CHECK_ACCESS_TYPE_IN_RANGE`?

Reply:
(Same on line 870 as well)